docs/conventions.md

```diff
@@ -0,0 +1,65 @@
+# Conventions
+
+## Contents
+
+- [Naming](#naming)
+- [Commit Messages and Pull Requests Names](#commit-messages-and-pull-request-names)
+- [Reviews](#reviews)
+- [Versioning](#versioning)
+
+## Naming
+
+We follow the following styles for the different languages:
+- TypeScript/JSX: [Airbnb Style](https://github.com/airbnb/javascript/tree/master/react)
+- Dart: [Effective Dart](https://dart.dev/effective-dart/style)
+- Kotlin: [Kotlin Style Guide](https://kotlinlang.org/docs/coding-conventions.html)
+
+## Commit Messages and Pull Request Names
+
+Commit messages should have the following schema:
+`<issue number>: Your commit message`, e.g. `1234: Add commit message documentation`
+
+Commit messages and PR names should be short but concise and explain what was done.
+Always use present tense (`Add` instead of `Added`).
+
+See [this guide](https://github.com/erlang/otp/wiki/Writing-good-commit-messages) for a general reference on how to
+write good commit messages and pull request names.
+
+## Branch Names
+
+Branch names should use lower-kebab-case and be prefixed with the issue number:
+`<issue number>-branch-name`, e.g. `1234-commit-message-documentation`
+
+_Note: Branch names (and PR names) do NOT have to match the issue title. Instead, try to be short and concise to focus
+on the actual work done._
+
+## Reviews
+
+We use the following emoji code for reviewing:
+
+- :+1: or `:+1:` This is great! It always feels good when somebody likes your work. Show them!
```

Comment:
we already have this contributing part in the general readme. please delete it there to avoid duplication